src/helm/config/model_deployments.yaml

```diff
@@ -195,6 +195,20 @@ model_deployments:
     client_spec:
       class_name: "helm.clients.anthropic_client.AnthropicClient"
 
+  - name: anthropic/claude-3-sonnet-20240229
+    model_name: anthropic/claude-3-sonnet-20240229
+    tokenizer_name: anthropic/claude
+    max_sequence_length: 200000
+    client_spec:
+      class_name: "helm.clients.anthropic_client.AnthropicMessagesClient"
+
+  - name: anthropic/claude-3-opus-20240229
+    model_name: anthropic/claude-3-opus-20240229
+    tokenizer_name: anthropic/claude
+    max_sequence_length: 200000
+    client_spec:
+      class_name: "helm.clients.anthropic_client.AnthropicMessagesClient"
```

Comment:
What about haiku?

Reply:
No Haiku because "Haiku will be available soon." ([source](https://www.anthropic.com/news/claude-3-family))